### PR TITLE
Add generator usage docs

### PR DESCRIPTION
**Pull request by redonkulus**
@Vijar @jedireza

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -17,9 +34,16 @@ If you want to jump right in to the individual libraries check them out:
...
+Fluxible is constructed of smaller core components that can be used separately if needed.
 
 - [dispatchr](https://github.com/yahoo/dispatchr) - Dispatchr's main goal is to facilitate server-side rendering of Flux applications while also working on the client-side to encourage code reuse. In order to isolate stores between requests on the server-side, we have opted to instantiate the dispatcher and stores classes per request.
 - [fetchr](https://github.com/yahoo/fetchr) - Fetchr provides an abstraction so that you can fetch (CRUD) your data in your stores using the same exact syntax on server and client side.
 - [routr](https://github.com/yahoo/routr) - Routr is an implementation of router-related functionalities that can be used for both server and client. This library does not use callbacks for routes, instead just mapping them to string names that can be used as application state and used within your application later. For instance in Flux, the current route would be held as state in a store.
 - [flux-router-component](https://github.com/yahoo/flux-router-component) - Provides navigational React components and router mixin for applications built with Flux architecture.
+
+
+
+
```
Comment (jedireza) [inline docs/quick-start.md]: Do we need all these new lines?

Comment (Vijar) [inline docs/quick-start.md]: Don't they all collapse to one line in the output anyway?

File: docs/quick-start.md [hunk 2]
```diff
@@ -17,9 +34,16 @@ If you want to jump right in to the individual libraries check them out:
 - [fluxible-plugin-fetchr](https://github.com/yahoo/fluxible-plugin-fetchr) - Provides isomorphic RESTful service access to your Fluxible application using Fetchr.
 - [fluxible-plugin-routr](https://github.com/yahoo/fluxible-plugin-routr) - Provides routing methods to your Fluxible application using Routr.
 
-### Libraries
+
+## Components
```
Comment (Vijar) [inline docs/quick-start.md]: There is ambiguity in this name. "Components" in the context of react usually gets me thinking of reusable react components.

Comment (redonkulus) [inline docs/quick-start.md]: Whats a better name? Core? Base? I dunno.

**Comment by redonkulus** [comment]
@jedireza @Vijar updated

**Comment by Vijar** [comment]
:ship:

**Comment by jedireza** [comment]
:shipit:

**Pull request merged**